extensions/big-data-cluster/package.json

```diff
@@ -187,6 +187,6 @@
   },
   "devDependencies": {
     "@types/request": "^2.48.3",
-    "vscode": "^1.1.36"
+		"@types/vscode": "^1.53.0"
```

Comment:
We probably shouldn't even be using the types package here - instead just having these in-repo extensions reference the vscode.d.ts directly (which most should already be set up to do)

Reply:
yeah, that makes sense.  I am trying to make a minimal fix to resolve the break, but certainly these can be refactored.

Reply:
That's fair - and for the release branch that's fine. Let me know if you want help with doing the same in main since I care more about doing it "correctly" there. I'd be glad to help out.

Reply:
sounds good, if you'd like to prepare a fix for main please proceed.  I'm currently only focusing on the release branch since this is blocking the Feb hotfix.  I was planning to look at main subsequently.